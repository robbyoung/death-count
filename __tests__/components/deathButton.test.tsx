import React from 'react';
import renderer from 'react-test-renderer';
import DeathButton from '../../app/components/deathButton';

describe('Death Button Component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<DeathButton
            onPress={() => console.log('Button Pressed')}
        />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});