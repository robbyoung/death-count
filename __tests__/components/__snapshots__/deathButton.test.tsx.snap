// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Death Button Component renders correctly 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "flexDirection": "row",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "color": undefined,
          },
          Object {
            "backgroundColor": "#e84641",
            "borderRadius": 110,
            "color": "#ffffff",
            "fontSize": 120,
            "height": 220,
            "textAlign": "center",
            "textAlignVertical": "center",
            "width": 220,
          },
          Object {
            "fontFamily": "FontAwesome5FreeSolid",
          },
        ]
      }
    >
      ï•Œ
    </Text>
  </View>
</View>
`;
